### PR TITLE
Pointers issue

### PR DESCRIPTION
**Pull request by Neofine**

### CODE REVIEW

File: frame/request/batch.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ type BatchQuery struct {
 	Values   []frame.Value
 }
 
-func (q BatchQuery) WriteTo(b *frame.Buffer, name bool) { // nolint:gocritic
+func (q *BatchQuery) WriteTo(b *frame.Buffer, name bool) { // nolint:gocritic
```
Comment (Choraden) [inline frame/request/batch.go:54]: Passing by pointer should resolve gocritic issue (#71) as well, so nolint comments can be deleted.

Comment (Neofine) [reply frame/request/batch.go:54]: Great! Will change it right away, thx!

Comment (Choraden) [reply frame/request/batch.go:54]: Actually there is one more gocritic nolint in buffer.go so don't link this issue until this is fixed.

**Comment by Neofine** [comment]
I am closing this PR as some refactoring was done.

**Pull request closed without merging**